apps/emqx_authn/src/simple_authn/emqx_authn_mongodb.erl

```diff
@@ -146,19 +146,19 @@ authenticate(#{password := Password} = Credential,
     case emqx_resource:query(Unique, {find_one, Collection, Selector2, #{}}) of
         undefined -> ignore;
         {error, Reason} ->
-            ?SLOG(error, #{msg => "query failed",
-                           unique => Unique,
+            ?SLOG(error, #{msg => "mongodb_query_failed",
```

Comment:
if this error is to be ignored, maybe should be at warning level instead.

Reply:
The purpose of the error indicates that there is a problem with the current external resource. The ignoring is to ensure authentication keep working. There will not be a problem with one instance that will cause the entire authentication feature to be unavailable.